enhancements/accelerators/out-of-tree-drivers.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Supporting out-of-tree drivers on BareMetal & OpenShift
+  - "@zvonkok"
+reviewers:
+  - TBD
+  - "@mrunalp"
+approvers:
+  - TBD
+  - "@mrunalp"
+creation-date: 2020-04-03
+last-updated: 2020-06-02
+status: provisional
+see-also:
+  - "/enhancements/TODO.md"  
+replaces:
+  - "/enhancements/TODO.md"
+superseded-by:
+  - "/enhancements/TODO.md"
+---
+
+# Supporting out-of-tree drivers on BareMetal & OpenShift
```

Comment:
Does this mean RHEL BareMetal installation and OpenShift installs? Or OpenShift Baremetal and Cloud installs?

Reply:
```suggestion
# Supporting out-of-tree drivers on OpenShift
```